lib/file_store/s3_store.rb

```diff
@@ -162,10 +196,14 @@ def path_for(upload)
 
     def url_for(upload, force_download: false)
       upload.secure? || force_download ?
-        presigned_url(get_upload_key(upload), force_download: force_download, filename: upload.original_filename) :
+        presigned_get_url(get_upload_key(upload), force_download: force_download, filename: upload.original_filename) :
         upload.url
     end
 
+    def path_from_url(url)
+      URI.parse(url).path[1..-1]
```

Comment:
```suggestion
      URI.parse(url).path.delete_prefix('/')
```
Again this is minor but I find `delete_prefix` clearer with regards to intent.

Reply:
Much better thanks, I forget about these nice little ruby helpers a lot